src/template_compiler.erl

```diff
@@ -301,7 +301,8 @@ translations(Filename) ->
             end;
         {error, _} = Error ->
             Error
-    end.
+    end,
+    {Filename, Translations}.
```

Comment:
Why would you return the filename? It is given as an input to the function, so returning it as well seems unnecessary.